src/index.html

```diff
@@ -2,11 +2,27 @@
 <html>
     <head>
         <title>IssueAi</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
-        <link href="https://fonts.googleapis.com/css?family=Lato:300&display=swap" rel="stylesheet">
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta
+            name="description"
+            content="O Issue Ai cria um espaço de visibilidade para os projetos open source de Computação@UFCG."
+        />
+        <meta name="theme-color" content="#FFFFFF" />
+        <link
+            rel="stylesheet"
+            href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+            integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+            crossorigin="anonymous"
+        />
+        <link
+            href="https://fonts.googleapis.com/css?family=Lato:300&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="manifest" href="/manifest.json" />
     </head>
     <body>
         <div id="app"></div>
+        <script src="./swRegister.js"></script>
```

Comment:
actually is 
```suggestion
        <script src="../public/swRegister.js"></script>
```

Reply:
Não vai funcionar assim, o que ele vai fazer na real é isso `https://issueai.opendevufcg.org/swRegister.js`

Eu coloquei na pasta pública justamente porque o webpack vai começar a servir esse arquivo na url.

Se eu fizer da forma que tu diz, não vai funcionar pro manifest.json e o serviceworker;

Eu coloquei esse script tbm pra ficar mais fácil de entender.